src/com/palmergames/bukkit/towny/object/Town.java

```diff
@@ -1377,4 +1377,8 @@ public boolean collect(double amount, String reason) throws EconomyException {
 	public String getTownBoard() {
 		return getBoard();
 	}
+
+	public double getEstimatedValueOfTown() {
```

Comment:
I feel like the method is better suited as a static convenience method in a util class rather than putting it directly in the town class.

Reply:
Ok I will move it

Reply:
Done